# Rhea Nibert, CIS4340 Lab 02
I initially uploaded the noncompliant code examples to the "edits" branch, fixed them to include the compliant solution, then merged these changes with "master". View the full commit history to see the initial noncompliant examples.
